README.rst

```diff
@@ -31,6 +36,11 @@ Features
 - Installed binaries are symbolically linked for better cross-platform usage.
   For example (on Windows), the ``clang-tidy-13.exe`` binary executable can
   also be invoked with the symbolic link titled ``clang-tidy.exe``
+
+  .. note::
+      To create symbolic links on Windows, you must enable developer mode
+      from the Windows settings under "Privacy & security" > "For developers"
+      category.
```

Comment:
Do we also need to update the Usage section? 

```bash
usage: clang-tools [-h] [-i INSTALL] [-d DIRECTORY] [-f] [-b]

optional arguments:
  -h, --help            show this help message and exit
  -i INSTALL, --install INSTALL
                        Install clang-tools with specific version. default is 13.
  -d DIRECTORY, --directory DIRECTORY
                        The directory where is the clang-tools install.
  -f                    Force overwriting the symlink to the installed binary. This will only overwrite an existing
                        symlink.
  -b, --no-progress-bar
                        Do not display a progress bar for downloads.
```

Reply:
I could set up the docs to auto-generate the doc about CLI options [like I did for the cpp-linter repo](https://cpp-linter.github.io/cpp-linter/cli_args.html). Now that I think about it, I haven't updated the logo and favicon for these docs.

Reply:
what are your thoughts on https://github.com/cpp-linter/clang-tools-pip/issues/26#issuecomment-1229168829?

Reply:
And it seems I made grammar mistakes in the description

Before

```
Install clang-tools with specific version
The directory where is the clang-tools install.
```
After

```
Install clang-tools with a specific version
The directory where is the clang-tools installed.
```

Reply:
Technically, the verb should go between the subject of the preposition. And the preposition "with" should be "of".
```
Install clang-tools [of || about] a specific version
The directory where the clang-tools [are] installed.
```
Sorry, I used to tutor English to foreign college students, so I learned to respect the author's voice despite personal experience or understanding.

Reply:
Please change it according to your thoughts. I 100% follow you ...

Reply:
ok I updated the grammar and the  `-i` default value. The CLI args are now automatically documented, so this section in readme is replaced with a link to the new generated document.

FYI, the new link in the readme will be broken until this is merged and the hosted docs are updated.